pandas/tests/indexing/test_loc.py

```diff
@@ -1130,3 +1130,17 @@ def test_loc_with_period_index_indexer():
     tm.assert_frame_equal(df, df.loc[list(idx)])
     tm.assert_frame_equal(df.iloc[0:5], df.loc[idx[0:5]])
     tm.assert_frame_equal(df, df.loc[list(idx)])
+
+
+def test_loc_replace_subset_with_subset():
+    # GH#36424 Should raise a warning
+    df1 = pd.DataFrame(
+        data=np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]]),
+        columns=['A', 'B', 'C'],
+        index=[0, 0, 1]
```

Comment:
You need to run black on this file, probably it doesn't like the single quotes and lack of trailing comma